cranelift/filetests/filetests/runtests/br.clif

```diff
@@ -156,3 +157,4 @@ block2:
 ; run: %brnz_i8(0) == 0
 ; run: %brnz_i8(1) == 1
 ; run: %brnz_i8(-1) == 1
+; run: %brnz_i8(97) == 1
```

Comment:
while here, a useful runtest to add might be a very slight variation on this one that intentionally tries to have nonzero bits in the upper undef bits of the `i8`-in-register, to make sure we're really testing just the low 8 bits; for example, by adding and wrapping:

```
block0(v0: i8):
  v1 = iconst.i8 255
  v2 = iadd.i8 v0, v1
  brz v2, ...
```

then it should branch for input arg `v0 = 1` even though the register will likely contain `256`...

Reply:
That's a great idea, I'll add that test!